From ed419b1d8c09af4660e89a4e1a28aaa7739f501b Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Mon, 23 Jan 2023 23:50:22 -0500
Subject: mingw GCC option O3 has bugs too often;

use O2 instead. The problem is not in GCC; but,
the libraries that use it tend to use things that
are undefined behavior reveiled by O3 option.
---
 Make.inc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Make.inc b/Make.inc
index 4719a3d49b..996432c3f5 100644
--- a/Make.inc
+++ b/Make.inc
@@ -526,7 +526,7 @@ ifneq ($(OS), WINNT)
 JCXXFLAGS += -pedantic
 endif
 DEBUGFLAGS := -O0 -ggdb2 -DJL_DEBUG_BUILD -fstack-protector
-SHIPFLAGS := -O3 -ggdb2 -falign-functions
+SHIPFLAGS := -O2 -ggdb2 -falign-functions
 endif
 
 ifeq ($(USECLANG),1)
-- 
